cli/cmd/common.go

```diff
@@ -35,7 +35,7 @@ func startProxyReportErrorIfAny(kubernetesProvider *kubernetes.Provider, ctx con
 		return
 	}
 
-	apiProvider = apiserver.NewProviderWithoutRetries(GetApiServerUrl(), time.Second) // short check for proxy
```

Comment:
why do we need 3 retries.. 
I remember that I implement the NewProviderWithoutRetries and there was a reason for it (and now if you are removing this (which is the only place that calls this method - it should be removed)